src/plugins/console/public/styles/_app.scss

```diff
@@ -10,6 +10,15 @@
 
 .consoleContainer {
   padding: $euiSizeS;
+
+  &--embeddable {
+    padding: 0;
+    gap: 0;
+  }
+}
+
+.consoleTabs {
+  padding: 0 $euiSizeS 0 $euiSizeS;
```

Comment:
nit: this can be shortened to...

```padding: 0 $euiSizeS;```

Reply:
Thanks! Fixed in [eb0b1d8](https://github.com/elastic/kibana/pull/190361/commits/eb0b1d893a1a029f80c79871b480c075d36c13e2).